app/marketing/mails.py

```diff
@@ -416,3 +416,20 @@ def setup_lang(to_email):
     if user and hasattr(user, 'profile'):
         preferred_language = user.profile.get_profile_preferred_language()
         translation.activate(preferred_language)
+
+
+def nth_day_email_campaign(nth, subscriber):
+    # take first part of email as default first name
+    firstname = subscriber.email.split('@')[0]
+
+    if subscriber.profile and subscriber.profile.user and subscriber.profile.user.first_name:
+        firstname = subscriber.profile.user.first_name
+
+    cur_language = translation.get_language()
+    try:
+        setup_lang(subscriber.email)
+        from_email = settings.CONTACT_EMAIL
+        html, text, subject = render_nth_day_email_campaign(nth, firstname)
```

Comment:
can you wrap this in a `should_suppress_notification_email()` call

Reply:
do you mean this

Reply:
yes that would be acceptable